craft_application/git/_git_repo.py

```diff
@@ -193,6 +231,65 @@ def _init_repo(self) -> None:
                 f"Could not initialize a git repository in {str(self.path)!r}."
             ) from error
 
+    def remote_exist(self, remote_name: str) -> bool:
```

Comment:
```suggestion
    def remote_exists(self, remote_name: str) -> bool:
```

Reply:
Done in https://github.com/canonical/craft-application/pull/396/commits/7d210e4d46d63bf131a38f3f583a6aaa31202618